### PR TITLE
feat: add Dockerfile

### PR DESCRIPTION
**Pull request by ashishbhate**

### CODE REVIEW

**Comment by ashishbhate** [comment]
Should I add a hadolint travis file as well?

**Comment by ferhatelmas** [comment]
@ashishbhate yes but could be a next PR.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM golang:1.12.2 as builder
+
+RUN go get -u github.com/client9/misspell/cmd/misspell
+
+FROM golang:1.12.2
```
Comment (ferhatelmas) [inline Dockerfile]: we don't need go anymore

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM golang:1.12.2 as builder
```
Comment (ferhatelmas) [inline Dockerfile]: we can use alpine version

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+RUN go get -u github.com/client9/misspell/cmd/misspell
+
+FROM golang:1.12.2
+COPY --from=builder /go/bin/* /usr/bin/
+
+WORKDIR /go
```
Comment (ferhatelmas) [inline Dockerfile]: specify entrypoint, misspell should be independent from go

**Comment by ashishbhate** [comment]
@ferhatelmas updated

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,10 @@
+FROM alpine:3.9
+
+LABEL maintainer="support@agflow.com" \
```
Comment (ferhatelmas) [inline Dockerfile]: `dev@agflow.com` is better.

**Pull request merged**